MANIFEST.in

```diff
@@ -1 +1,2 @@
-recursive-include wof/flask/templates *
\ No newline at end of file
+recursive-include wof/flask/templates *
+recursive-include wof/examples/flask/odm2/timeseries *
```

Comment:
I move the examples directory inside the `wof` module to package it as a sub-module of `wof` instead of a first class module named `examples` that can be easily overwriting if another package does the same.